content/post/2023/captioning-tools-extravaganza.md

```diff
@@ -0,0 +1,127 @@
+
+### General Thoughts
+
+First, I'd like to call out my operating context here: I'm a Midwestern American white male, working on teams primarily composed of the same. This, unfortunately, is the Platonic ideal for automatic speech to text processing. Depending on your accent, or your teammates', you may have radically different experiences than mine.
+
+Some tools are able to both transcribe audio **and** translate it to another language at the same time. However, as I've never used this capability, I'm unable to say how well it works.
+
+It's also important to emphasize that these tools are not perfect, and sometimes misunderstand what you say - this is pretty common for me because I have a strong "deaf" accent and don't sound like most other people (funnily, I'm occasionally asked if I'm British, so I guess that's how my accent comes across to American ears). One reason I mention this is because many of these tools offer profanity filters, and I've occasionally had them think I was swearing when I wasn't. This is something I wanted to call out; from a speaker's standpoint, if you're not accustomed to this and aren't expecting it, or expecting the audience to react to it, it might be a little surprising.
+
+> A conference I regularly attend - Stir Trek in Columbus, Ohio - runs a series of repeating announcements on the projector screens before each talk. Including a brief educational slide about automatic captions might be a _great_ idea here.
+
+With all of that said, let's examine options across various platforms. I will note if options are free or paid, but since prices are always changing, I'm not going to make an effort to call those out specifically.
+
+### As a Presenter, I want captioning, so that my talk is accessible
+
+Generally, the usual advice for speaking applies. Speak as clearly as you can, avoid cross-talk if you're co-presenting with someone. If you're doing a Q&A session, it's helpful to repeat the question back to make sure it shows up in the captioning feed.
+
+If you're able, turn on captions while you're doing practice runs for the talk. This should give you a sense of how accurate these tools are and how they sometimes misunderstand what you say. If your talk uses uncommon tech jargon, it's helpful to know how it may come up on screen.
+
+#### Windows Live Captions
+- **Free, requires Windows 11 22H2 or newer, works offline**
+
+_My thoughts_: Quick and pretty accurate, deals well with accents. Can be a little glitchy with repeating fragments of a sentence that's not finalized. Includes a profanity filter. I use this heavily every single day on both my work and home machines, and it's tremendously useful. This requires downloading several hundred megabytes of supporting files, so be sure to set it up ahead of time.
```

Comment:
I would remove "my thoughts" everywhere. It's a blog post, and you are writing this in the first person, so it's clear that this is your opinion. Don't think that spelling it out is necessary. 
```suggestion
Quick and pretty accurate, deals well with accents. Can be a little glitchy with repeating fragments of a sentence that's not finalized. Includes a profanity filter. I use this heavily every single day on both my work and home machines, and it's tremendously useful. This requires downloading several hundred megabytes of supporting files, so be sure to set it up ahead of time.
```